### ISSUE TITLE
Unauthorized bij lijsten

### ISSUE THREAD

**Issue by HeinPauwelyn** [WatchFriends/Web#179]
Kan dat zijn als je nu niet geregistreerd bent, dat je geen API request kunt doen naar `/api/list`. Je krijgt continue een Unauthorized terug. 

Dit is niet de bedoeling van de lijsten op de explorer pagina ~~en de home pagina van de Android app~~. Ik wil graag hebben dat nieuwe, niet geregistreerde gebruikers ook een aantal lijsten kunnen bekijken zodat ze de site al wat leren kennen.

**Comment by ExCellRaD** [comment]
Ja, alle api requests zijn geautoriseerd nu. Het is best om nu gewoon nog alles te testen terwijl je ingelogd bent, en dat we dan later kijken welke routes niet geautoriseerd moeten zijn.
